webpack/webpack.config.babel.js

```diff
@@ -0,0 +1,67 @@
+import autoprefixer from 'autoprefixer';
+import colorLighten from 'less-color-lighten';
+import path from 'path';
+
+function absPath() {
+  let args = [].slice.apply(arguments);
+  args.unshift(__dirname, '..');
+
+  return path.resolve.apply(path.resolve, args);
+}
+
+// Can override this with npm config set dcos-ui:external_plugins ../some/relative/path/to/repo
+let pluginsDir = absPath(process.env.npm_package_config_external_plugins || 'plugins');
```

Comment:
Could you suggest a default configuration so it won't fail to build by default? Most people who get started won't have an external plugins directory. I updated the name of the variable to be more explicit!